src/base/utils/fs.cpp

```diff
@@ -518,3 +518,10 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+QString Utils::Fs::tempPath()
+{
+    static const QString path = QDir::tempPath() + "/.qBittorrent/";
+    QDir().mkdir(path);
```

Comment:
> I tried that before, however I realized that the temp folder could be deleted when qbt is running,

This may very well happen with the current approach too. A lock file, maybe, if you want to take care of this situation, would be a better solution?